src/compiler/parser/index.js

```diff
@@ -476,7 +476,10 @@ function processSlot (el) {
         )
       }
       el.slotScope = slotScope || getAndRemoveAttr(el, 'slot-scope')
-    } else if ((slotScope = getAndRemoveAttr(el, 'slot-scope'))) {
+    } else if (
+        (slotScope = getAndRemoveAttr(el, 'slot-scope')) ||
+        (slotScope = getAndRemoveAttr(el, 'slot-scope')) === ''
```

Comment:
shouldn't we be able to do `slotScope === ''` as the second check?

Reply:
I wonder if we could only check against `!= null`

Reply:
Yes, we could do that as the second check, I don't think null is the only falsy value we should check against @posva

Reply:
it's `null` and `undefined`, but yeah, I didn't check further, just dropped the idea haha